packages/ui-core/src/components/TextField/TextField.tsx

```diff
@@ -170,16 +209,22 @@ const TextField: React.FC<ITextFieldProps> = ({
         onBlur && onBlur(e);
     }, [onBlur]);
 
+    const getTextareaMod = () => (
+        textarea === TextareaTypes.FLEXIBLE ? TextareaTypes.FLEXIBLE : TextareaTypes.FIXED
+    );
+
+    const isScrolling = initialTextareaHeight === TEXTAREA_MAX_HEIGHT || isTextareaResized;
```

Comment:
Возможно
```suggestion
    const isScrolling = initialTextareaHeight >= TEXTAREA_MAX_HEIGHT || isTextareaResized;
```
Т.к. при каких-то будущих изменениях высоты строк или поля легко забыть про кейс с ресайзом. А так хоть какая-то защита.